tests/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ set_target_properties(unittests
   )
 
 target_link_libraries(unittests
-    PRIVATE morphio_shared HighFive
+    PRIVATE morphio_static HighFive
```

Comment:
why can't they be shared?

Reply:
'unittests' was failing to link with morphio lib. It was looking for 'morphio.lib' while the shared lib was called 'morphio.dll'. There is probably a better way to fix that but I'm A1 in CMAKE so I don't know.